templates/gameLog.html

```diff
@@ -76,7 +76,8 @@
                 <td data-bind="text: stl"></td>
                 <td data-bind="text: blk"></td>
                 <td data-bind="text: pf"></td>
-                <td data-bind="text: pts"></td>
+                  <td data-bind="text: pts"></td>
+                  <td data-bind="round: [pts() + 0.4  * fg() - 0.7 * fga() - 0.4*(fta() - ft()) + 0.7 * orb() + 0.3 * (trb() - orb())  + stl() + 0.7 * ast() + 0.7 * blk() - 0.4 * pf() - tov(),0]"></td>
```

Comment:
This formula should be abstracted somehow so it doesn't have to be copy/pasted in the two files here (and other files later). I'm not sure if it would be better to pre-calculate before getting to the template or calculate with a Knockout binding in the template. What do you think?